tests/core/test_lightning_optimizer.py

```diff
@@ -98,20 +96,24 @@ def training_step(self, batch, batch_idx, optimizer_idx=None):
 
             output = self.layer(batch)
             loss_1 = self.loss(batch, output)
-            self.manual_backward(loss_1, opt_1)
+            self.manual_backward(loss_1)
             opt_1.step()
+            opt_1.zero_grad()
 
             def closure():
                 output = self.layer(batch)
                 loss_2 = self.loss(batch, output)
-                self.manual_backward(loss_2, opt_2)
+                self.manual_backward(loss_2)
```

Comment:
why do you need to define it as a function when you just call it in place?

Reply:
Done !